src/index.tsx

```diff
@@ -376,7 +376,7 @@ export function formAtom<Fields extends FormFields>(
     });
   }
 
-  return atom(formAtoms);
+  return atom({ ...formAtoms, _validateFields: validateFields });
```

Comment:
Can you add this property to the object on L359 instead of using a destructuring assignment? Otherwise lgtm

Reply:
This is intentional, as the `validateFields` is a plain function, not an atom.

Otherwise the debugLabel on L375 would fail.

Reply:
Got it, but can't you still assign it without destructuring first?
```ts
formAtoms._validateFields = validateFields
```

I'm just not a fan of the extra work

Reply:
It requires me to add some ugly explicit casts:

```ts
  (formAtoms as Record<string, unknown>)._validateFields = validateFields;

  return atom(formAtoms) as unknown as FormAtom<Fields>;
```
hm